docs/system-admin-guide/manage-work-packages/work-package-status/README.md

```diff
@@ -12,24 +12,42 @@ Work packages can have various status depending on their types, e.g. tasks can h
 
 The status can be set at the top of the work package details view or in the table view in the corresponding column.
 
-![Work packages status dropdown menu in OpenProject](openproject_system_guide_create_wp_stati_dropdown.png)The status field can be configured under *Administration ->* *Work packages* -> *Status*. You will see the list of all the existing work package status in the system. You can add new status or change the name and order of existing status.
+![Work packages status dropdown menu in OpenProject](openproject_system_guide_create_wp_stati_dropdown.png)
+
+The status field can be configured under *Administration ->* *Work packages* -> *Status*. You will see the list of all the existing work package stati in the system. You can add new status or change the name and order of existing status.
```

Comment:
```suggestion
The status field can be configured under *Administration ->* *Work packages* -> *Status*. You will see the list of all the existing work package statuses in the system. You can add new statuses or change the name and order of existing statuses.
```

_stati_ sounds wrong. I think we use _statuses_ all over the application. 
https://english.stackexchange.com/questions/877/what-is-the-plural-form-of-status